### PR TITLE
Add notifications to twenty.js

### PR DESCRIPTION
**Pull request by deanylev**

### CODE REVIEW

File: public/js/twenty.js [hunk 1]
```diff
@@ -25,6 +29,31 @@ document.addEventListener('DOMContentLoaded', () => {
...
+        return;
+      }
+
+      message = 'Time to look away!';
+    } else {
+      if (stop) {
+        message = 'Time to stand up!';
+      } else {
+        message = 'Time to sit down!';
+      }
```
Comment (DuBistKomisch) [inline public/js/twenty.js]: can you make it play "up" and "down" from https://www.youtube.com/watch?v=qJdMjRHRLfg?

File: public/js/twenty.js [hunk 2]
```diff
@@ -25,6 +29,31 @@ document.addEventListener('DOMContentLoaded', () => {
...
+
+    let message;
+
+    if (mode === 'twenty') {
+      if (!stop) {
+        // can't see the notification if they're looking away!
+        return;
+      }
+
+      message = 'Time to look away!';
```
Comment (DuBistKomisch) [inline public/js/twenty.js]: might be better to have these strings as data like `limits`, maybe a copy pasted `notifications` structure? then you could just look it up and handle everything in one way

File: public/js/twenty.js [hunk 3]
```diff
@@ -12,6 +12,10 @@ let mode = 'twenty';
 let stop = true;
 let count = 0;
 
+if (Notification.permission === 'default') {
```
Comment (DuBistKomisch) [inline public/js/twenty.js:24]: what a weird API

Comment (deanylev) [reply public/js/twenty.js:24]: Very weird

File: public/js/twenty.js (hunk 2 above)
Comment (deanylev) [reply public/js/twenty.js]: Done, I also made it not notify you on startup / if you manually change the mode by clicking

**Review by DuBistKomisch — APPROVED** [review]

**Pull request merged**